compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -4134,9 +4143,9 @@ private void startContextIfRequired(ParserRuleContext currentCtx) {
             case TYPE_DESC_IN_EXPRESSION:
             case TYPE_DESC_IN_STREAM_TYPE_DESC:
             case TYPE_DESC_IN_PARENTHESIS:
-            case TYPE_DESC_IN_TUPLE:
```

Comment:
In the parser when we call, ` parseTypeDescriptor(ParserRuleContext.TYPE_DESC_IN_TUPLE);` we start a context for `TYPE_DESC_IN_TUPLE` inside. Therefore we need it in the error handler as well.